plugins/workspace-minimap/src/focus_region.ts

```diff
@@ -45,6 +45,10 @@ export class FocusRegion {
     private minimapWorkspace: Blockly.WorkspaceSvg,
   ) {
     this.id = String(Math.random()).substring(2);
+    this.onChangeWrapper = (e: Blockly.Events.Abstract) => {}; 
+    this.svgGroup = document.createElementNS('http://www.w3.org/2000/svg', 'g');
+    this.rect = document.createElementNS('http://www.w3.org/2000/svg', 'rect');
+    this.background = document.createElementNS('http://www.w3.org/2000/svg', 'rect'); 
```

Comment:
If these are `| null` we don't need to assign them in the constructor. Can you remove these?